src/esw/brushed_motors/Controller.cpp

```diff
@@ -76,12 +101,84 @@ void Controller::moveOpenLoop(float input) {
 
         I2C::transact(deviceAddress, motorIDRegMask | OPEN_PLUS_OP, OPEN_PLUS_WB,
                       OPEN_PLUS_RB, buffer, UINT8_POINTER_T(&angle));
-        currentAngle = (float) ((((float) angle / quadCPR) * 2 * M_PI) - M_PI);
+        currentAngle = (float) (((float) angle / quadCPR) * 2 * M_PI);
```

Comment:
It's not extraneous, it is needed to pass style check since M_PI is a double.

 error: narrowing conversion from 'double' to 'float' [cppcoreguidelines-narrowing-conversions,-warnings-as-errors]

Reply:
oh okay makes sense